luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -203,16 +205,11 @@ def _genotype_fields(self):
             'num_alt': self._num_alt(is_called)
         }
 
-    # NB: override this function here to mimic the existing null handling behavior.
-    def _genotype_filter_samples(self, filter_):
-        samples = super()._genotype_filter_samples(filter_)
-        return hl.or_missing(hl.len(samples) > 0, samples)            
```

Comment:
I'd added this so that we return missing values.... now we return the empty set.